asyncapi/asyncapi.yml

```diff
@@ -251,33 +250,31 @@ channels:
             retain: true
   pe/dpi_ack:
```

Comment:
@eivindga should we change this to `pe/dpi/ack`?

Reply:
yes